doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md

```diff
@@ -10,53 +10,61 @@ sort_idx: 40
+You can use the necessary APIs to attach/detach endpoints to/from owners.
 
+To do this, you can use either [owner access token](#using-the-owner-access-token), or [endpoint access token](#using-the-endpoint-access-token).
 
-# Owner access token flow #
-In the *owner access token flow*, the owner authenticates himself in an external authentication system and obtains the *access token*. 
-The owner performs this authentication from the endpoint which is due to be registered with him in the Kaa instance. 
-Then, Kaa SDK transfers this token to the Kaa cluster over a secure channel. The Kaa cluster verifies the access token and attaches the endpoint to the owner.
+## Using the owner access token
 
+As an owner, you can authenticate using an external authentication system and obtain your _access token_.
+You need to perform this authentication from the endpoint that is duly registered in the [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment).
+Your [endpoint SDK]({{root_url}}Glossary/#endpoint-sdk) sends this token to the [Kaa cluster]({{root_url}}Glossary/#kaa-cluster) over a secure channel.
+The Kaa cluster verifies the access token and attaches the endpoint to the owner.
 
 ![Owner access token flow](owner_attach_2.png "Owner access token flow")
 
+### Owner verifiers
 
-## Owner verifiers ##
-The owner verification is handled by specific server components called owner verifiers. 
-There are several default owner verifier implementations that are available out of the box for each Kaa installation. 
-This section contains general information about the architecture, configuration and administration of the default owner verifiers. 
-It is also possible to plug in [custom verifier implementations](#custom-owner-verifier). Each Kaa application can support multiple owner verifiers.
-You can add new verifier from Administration UI or with using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
-The following image example illustrates how to add new verifier from Administration UI.
+The owner verification is handled by specific server components called _owner verifiers_.
+There are several default implementations of the owner verifier available out of the box for each Kaa installation.
+This section contains general information about the architecture, configuration, and administration of the default owner verifiers.
+You can also plug in [custom verifier implementations](#custom-owner-verifier).
+One [Kaa application]({{root_url}}Glossary/#kaa-application) can support multiple owner verifiers.
+You can add a new verifier using the [Administration UI]({{root_url}}Glossary/#administration-ui) or [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
+
+You can add user verifiers using the **User verifier** page of your application.
```

Comment:
"You can, you can..."
Remove this line and reshuffle the previous sentence to first refer to REST API, and then the **User verifier** page of the Administration UI.